logstash_formatter/__init__.py

```diff
@@ -84,32 +84,28 @@ def format(self, record):
         except (KeyError, IndexError):
             pass
 
-        if 'msg' in fields:
-            fields.pop('msg')
-
-        if 'exc_info' in fields:
-            if fields['exc_info']:
-                formatted = tb.format_exception(*fields['exc_info'])
-                fields['exception'] = formatted
-            fields.pop('exc_info')
+        if fields.get('exc_info', None):
```

Comment:
You don't have to set `None` as a default here. `if fields.get('exc_info')` should return None if it's not there

```python
In [1]: fields = {}

In [2]: if fields.get('exc_info'):
   ...:     print 'yolo'
   ...:

In [3]: fields = {'exc_info': 'yolo'}

In [4]: if fields.get('exc_info'):
    print 'yolo'
   ...:
yolo

```